rllib/policy/tf_mixins.py

```diff
@@ -241,6 +248,14 @@ def target_q_func_vars(self):
             self._target_q_func_vars = self.target_model.variables()
         return self._target_q_func_vars
 
+    # Support both hard and soft sync.
+    def update_target(self, tau: int = None) -> None:
```

Comment:
why do you need this member function?
you can simply assign update_target_fn to self.update_target?

Reply:
This is how it was done before? I suspect it has something to do with eager vs. non-eager mode. I just replicated code and unified the internal logic.

Reply:
wait, before, we don't have
```
def update_target(...)
```
right?
we create the member function in ``__init__()``, and that's it?

Reply:
This is the current master for one of the torch TargetNetworkMixins (without tau) 
https://sourcegraph.com/github.com/ray-project/ray/-/blob/rllib/policy/torch_mixins.py

Reply:
We do have `update_target()`